client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -376,22 +376,10 @@ class CalypsoifyIframe extends Component<
 
 		if ( EditorActions.GetGutenboardingStatus === action ) {
 			const isGutenboarding = this.props.siteCreationFlow === 'gutenboarding';
-			const isSiteUnlaunched = this.props.isSiteUnlaunched;
-			const launchUrl = `${ window.location.origin }/start/launch-site?siteSlug=${ this.props.siteSlug }`;
-			const isNewLaunchMobile = config.isEnabled( 'gutenboarding/new-launch-mobile' ); // TODO: remove after ETK 2.8.6 is released
-			const isExperimental = config.isEnabled( 'gutenboarding/feature-picker' ); // TODO: remove after ETK 2.8.6 is released
-			const isPersistentLaunchButton = config.isEnabled( 'create/persistent-launch-button' ); // TODO: remove after ETK 2.8.6 is released
```

Comment:
Shouldn't we also cleanup the feature flags from config files?

Reply:
Okay so these 3 feature flags are now removed.